k8s/contiv-vpp/values.yaml

```diff
@@ -0,0 +1,73 @@
+contiv:
+  tcpStackDisabled: True
+  useTAPInterfaces: True
+  tapInterfaceVersion: 1
+  ipamConfig:
+    podSubnetCIDR: "10.1.0.0/16"
+    podNetworkPrefixLen: 24
+    vppHostSubnetCIDR: "172.30.0.0/16"
+    vppHostNetworkPrefixLen: "192.168.16.0/24"
+    vxlanCIDR: "192.168.30.0/24"
+    nodeInterconnectCIDR:
```

Comment:
why is nodeInterconnectCIDR empty?

Reply:
This was a recommendation from @nbartos so that NodeInterconnectDHCP would default to True. https://github.com/fattarsi/vpp/blob/vpp-helm-chart/k8s/contiv-vpp/templates/vpp.yaml#L35 - If specified or overridden, it would set to False.